src/collections/blog/2022/2022-06-27-converting-docker-compose-files-and-kubernetes-manifests-to-meshery-pattern-files/index.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: "Converting Docker Compose files and Kubernetes Manifests to Meshery Pattern Files"
+date: 2022-06-27 01:15:00 -0600
+author: Ruturaj Mohite
+description: Converting Docker Compose files and Kubernetes Manifests to Meshery Pattern Files
+thumbnail: ./thumbnail.png
+type: Blog
+product: Meshery
+category: "Meshery"
+tags:
+ - Projects
+ - Meshery
+ - Patterns
+published: true
+featured: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import Blockquote from "../../../../reusecore/Blockquote";
+
+import dashboardImg from "./dashboard.png";
+import applicationsPageImg from "./applications_page.png";
+import importAppImg from "./import_app.png";
+import clickDeployImg from "./click_deploy.png";
+import requestPayloadImg from "./request_payload.png";
+
+<BlogWrapper>
+
+<h3>Meshery Pattern Files</h3>
+<p><a href="/service-mesh-management/meshery">Meshery</a> uses <a href="/learn/service-mesh-books/service-mesh-patterns">Service Mesh Patterns</a> that act as configurations of service meshes or applications to be deployed through Meshery. These configurations are written in a YAML format in a pattern file. Here's a small example pattern file to deploy a demo deployment of Istio through Meshery:
```

Comment:
```suggestion
<p><Link href="/service-mesh-management/meshery">Meshery</a> uses <Link href="/learn/service-mesh-books/service-mesh-patterns">Service Mesh Patterns</a> that act as configurations of service meshes or applications to be deployed through Meshery. These configurations are written in a YAML format in a pattern file. Here's a small example pattern file to deploy a demo deployment of Istio through Meshery:
```

Reply:
Let's import and use the gatsby `Link` component for internal linking. Rest looks good.